build/build.generic.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+
+rem
+rem Parameters
```

Comment:
Since there are 5 (possibly more) arguments, I could at least `echo` the commands if there are NO arguments, but maybe replace the others with default values? I need to brush up on my batch skills, it's been a long time :)